solaris/solaris11/uninstall.sh

```diff
@@ -0,0 +1,22 @@
+#/bin/sh
+
+## Stop and remove application
+sudo /var/ossec/bin/ossec-control stop
+sudo rm -r /var/ossec*
+sudo rm /etc/ossec-init.conf
+
+## stop and unload dispatcher
+#sudo /bin/launchctl unload /Library/LaunchDaemons/com.wazuh.agent.plist
```

Comment:
Unnecessary line

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893